### PR TITLE
refactor(dsp-docs): clean up (DSP-1326)

### PR DESCRIPTION
**Pull request by flavens**
resolves DSP-1326

### CODE REVIEW

File: docs/community/product-updates.md [hunk 1]
```diff
@@ -6,11 +6,11 @@
 
 <https://github.com/dasch-swiss/knora-api/releases>
 
-## DSP-JS-LIB (previously knora-api-js-lib)
```
Comment (kilchenmann) [inline docs/community/product-updates.md:9]: Is the information in the brackets (previously ...) still relevant? The only one we should keep is `## DSP-API (previously Knora)` --> knora without "-api".

File: docs/community/product-updates.md [hunk 2]
```diff
@@ -6,11 +6,11 @@
...
+## DSP-JS (previously knora-api-js-lib)
 
 <https://github.com/dasch-swiss/knora-api-js-lib/releases>
```
Comment (kilchenmann) [inline docs/community/product-updates.md:11]: Outdated link; same for dsp-app

**Review by kilchenmann — COMMENTED** [review]
We should also update the READMEs and replace Knora by DSP-API etc. there too.

File: docs/community/product-updates.md (hunk 1 above)
Comment (flavens) [reply docs/community/product-updates.md:9]: done in d73b7d9

File: docs/community/product-updates.md (hunk 2 above)
Comment (flavens) [reply docs/community/product-updates.md]: done in d73b7d9

**Review by kilchenmann — APPROVED** [review]

**Pull request merged**